community/blog/2020-05-Azure-Dependency-Tracking-in-Observability-v0.2.0/README.md

```diff
@@ -0,0 +1,59 @@
+# Azure dependency tracking in Observability v0.2.0
+
+With the release of the next version of [Arcus.Observability v0.2.0](https://github.com/arcus-azure/arcus.observability/releases/tag/v0.2.0) the support for dependency tracking of Azure resources is greattly expanded.
+
+Come see what this new version has to offer!
+For more information, see the [offical docs](https://observability.arcus-azure.net/).
+
+## Azure IoT Hub dependency tracking
+
+We allow you to measure Azure IoT Hub dependencies using the `LogIotHubDependency` overload.
+Tracking this dependency, shows on the application map:
+
+![iot hub application map](media/iot-hub-application-map.png)
+
+## Azure Blob storage dependency tracking
+
+We allow you to measure Azure Blob storage dependencies using the `LogBlobStorageDependency` overload.
+Tracking this depedency, shows on the application map:
+
+![blob storage application map](media/blob-storage-application-map.png)
+
+## Azure Cosmos Db dependency tracking
+
+We allow you to measure Cosmos Db dependencies using the `LogCosmosDdbDependency` overload.
+Tracking this dependency, shows on the application map:
+
+![cosmos db application map](media/cosmos-db-application-map.png)
+
+## Azure Event Hubs dependency tracking
+
+We allow you to measure Event Hubs dependencies using the `LogEventHubsDependency` overload.
+Tracking this dependency, shows on the application map:
+
+![event hubs application map](media/event-hubs-application-map.png)
+
+## Azure Table storagedependency tracking
+
+We allow you to measure Table storage dependencies using the `LogTableStorageDependency` overload.
+Tracking this dependency, shows on the application map:
+
+![table storage application map](media/table-storage-application-map.png)
+
+## SQL dependency tracking
+
+Tracking SQL dependencies was already available starting from [version v0.1.0](https://github.com/arcus-azure/arcus.observability/releases/tag/v0.1.0), but this version also provides an additional feature to make tracking these kind of dependencies more consumer friendly.
+
+Instead of providing the server, database, ... separatly, this version allows you to pass along the SQL connection string and extract the required information from it.
```

Comment:
```suggestion
Instead of providing all the connection information by hand, we now allow you to pass the SQL connection string and we'll extract the required information from it.
```